### PR TITLE
Add elasticsearch create Index task 

### PR DESCRIPTION
**Pull request by Ryohu**
```
Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/fog-1.23.0/lib/fog/rackspace/mock_data.rb:42: warning: key "name" is duplicated and overwritten on line 80
rake aborted!
ActiveRecord::StatementInvalid: PG::UndefinedTable: ERROR:  relation "users" does not exist
LINE 1: SELECT  "users".* FROM "users"  ORDER BY "users"."id" ASC LI...
                               ^
: SELECT  "users".* FROM "users"  ORDER BY "users"."id" ASC LIMIT 1000
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/rack-mini-profiler-0.9.8/lib/patches/db/pg.rb:90:in `exec'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/rack-mini-profiler-0.9.8/lib/patches/db/pg.rb:90:in `async_exec'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:592:in `block in exec_no_cache'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract_adapter.rb:472:in `block in log'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract_adapter.rb:466:in `log'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:592:in `exec_no_cache'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:584:in `execute_and_clear'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql/database_statements.rb:160:in `exec_query'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:355:in `select'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:32:in `select_all'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/query_cache.rb:70:in `select_all'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/querying.rb:39:in `find_by_sql'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/relation.rb:639:in `exec_queries'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/relation.rb:515:in `load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/relation.rb:243:in `to_a'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/relation/batches.rb:117:in `find_in_batches'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/querying.rb:9:in `find_in_batches'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/elasticsearch-model-0.1.8/lib/elasticsearch/model/proxy.rb:81:in `method_missing'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/elasticsearch-model-0.1.8/lib/elasticsearch/model/adapters/active_record.rb:95:in `__find_in_batches'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/elasticsearch-model-0.1.8/lib/elasticsearch/model/importing.rb:122:in `import'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/elasticsearch-model-0.1.8/lib/elasticsearch/model.rb:115:in `import'
/Users/hashimotoryo/Documents/ruby_clone/stories/app/models/user.rb:92:in `<top (required)>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:457:in `load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:457:in `block in load_file'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:647:in `new_constants_in'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:456:in `load_file'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:354:in `require_or_load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:494:in `load_missing_constant'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:184:in `const_missing'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:261:in `const_get'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:261:in `block in constantize'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:259:in `each'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:259:in `inject'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:259:in `constantize'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:566:in `get'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:597:in `constantize'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise.rb:292:in `get'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:81:in `to'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:76:in `modules'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:93:in `routes'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:160:in `default_used_route'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:70:in `initialize'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise.rb:326:in `new'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise.rb:326:in `add_mapping'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/rails/routes.rb:238:in `block in devise_for'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/rails/routes.rb:237:in `each'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/rails/routes.rb:237:in `devise_for'
/Users/hashimotoryo/Documents/ruby_clone/stories/config/routes.rb:6:in `block in <top (required)>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/actionpack-4.2.5/lib/action_dispatch/routing/route_set.rb:434:in `instance_exec'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/actionpack-4.2.5/lib/action_dispatch/routing/route_set.rb:434:in `eval_block'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/actionpack-4.2.5/lib/action_dispatch/routing/route_set.rb:412:in `draw'
/Users/hashimotoryo/Documents/ruby_clone/stories/config/routes.rb:3:in `<top (required)>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:268:in `load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:268:in `block in load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:240:in `load_dependency'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:268:in `load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:40:in `block in load_paths'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:40:in `each'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:40:in `load_paths'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:16:in `reload!'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:26:in `block in updater'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/file_update_checker.rb:75:in `execute'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:27:in `updater'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:7:in `execute_if_updated'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/finisher.rb:69:in `block in <module:Finisher>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `instance_exec'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `run'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/initializable.rb:55:in `block in run_initializers'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/initializable.rb:54:in `run_initializers'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application.rb:352:in `initialize!'
/Users/hashimotoryo/Documents/ruby_clone/stories/config/environment.rb:5:in `<top (required)>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `require'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `block in require'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:240:in `load_dependency'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `require'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application.rb:328:in `require_environment!'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application.rb:457:in `block in run_tasks_blocks'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/bin/ruby_executable_hooks:15:in `eval'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/bin/ruby_executable_hooks:15:in `<main>'
PG::UndefinedTable: ERROR:  relation "users" does not exist
LINE 1: SELECT  "users".* FROM "users"  ORDER BY "users"."id" ASC LI...
                               ^
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/rack-mini-profiler-0.9.8/lib/patches/db/pg.rb:90:in `exec'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/rack-mini-profiler-0.9.8/lib/patches/db/pg.rb:90:in `async_exec'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:592:in `block in exec_no_cache'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract_adapter.rb:472:in `block in log'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/notifications/instrumenter.rb:20:in `instrument'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract_adapter.rb:466:in `log'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:592:in `exec_no_cache'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql_adapter.rb:584:in `execute_and_clear'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/postgresql/database_statements.rb:160:in `exec_query'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:355:in `select'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/database_statements.rb:32:in `select_all'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/connection_adapters/abstract/query_cache.rb:70:in `select_all'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/querying.rb:39:in `find_by_sql'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/relation.rb:639:in `exec_queries'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/relation.rb:515:in `load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/relation.rb:243:in `to_a'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/relation/batches.rb:117:in `find_in_batches'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activerecord-4.2.5/lib/active_record/querying.rb:9:in `find_in_batches'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/elasticsearch-model-0.1.8/lib/elasticsearch/model/proxy.rb:81:in `method_missing'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/elasticsearch-model-0.1.8/lib/elasticsearch/model/adapters/active_record.rb:95:in `__find_in_batches'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/elasticsearch-model-0.1.8/lib/elasticsearch/model/importing.rb:122:in `import'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/elasticsearch-model-0.1.8/lib/elasticsearch/model.rb:115:in `import'
/Users/hashimotoryo/Documents/ruby_clone/stories/app/models/user.rb:92:in `<top (required)>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:457:in `load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:457:in `block in load_file'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:647:in `new_constants_in'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:456:in `load_file'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:354:in `require_or_load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:494:in `load_missing_constant'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:184:in `const_missing'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:261:in `const_get'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:261:in `block in constantize'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:259:in `each'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:259:in `inject'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/inflector/methods.rb:259:in `constantize'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:566:in `get'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:597:in `constantize'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise.rb:292:in `get'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:81:in `to'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:76:in `modules'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:93:in `routes'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:160:in `default_used_route'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/mapping.rb:70:in `initialize'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise.rb:326:in `new'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise.rb:326:in `add_mapping'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/rails/routes.rb:238:in `block in devise_for'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/rails/routes.rb:237:in `each'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/devise-3.5.6/lib/devise/rails/routes.rb:237:in `devise_for'
/Users/hashimotoryo/Documents/ruby_clone/stories/config/routes.rb:6:in `block in <top (required)>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/actionpack-4.2.5/lib/action_dispatch/routing/route_set.rb:434:in `instance_exec'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/actionpack-4.2.5/lib/action_dispatch/routing/route_set.rb:434:in `eval_block'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/actionpack-4.2.5/lib/action_dispatch/routing/route_set.rb:412:in `draw'
/Users/hashimotoryo/Documents/ruby_clone/stories/config/routes.rb:3:in `<top (required)>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:268:in `load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:268:in `block in load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:240:in `load_dependency'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:268:in `load'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:40:in `block in load_paths'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:40:in `each'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:40:in `load_paths'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:16:in `reload!'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:26:in `block in updater'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/file_update_checker.rb:75:in `execute'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:27:in `updater'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/routes_reloader.rb:7:in `execute_if_updated'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application/finisher.rb:69:in `block in <module:Finisher>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `instance_exec'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/initializable.rb:30:in `run'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/initializable.rb:55:in `block in run_initializers'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/initializable.rb:54:in `run_initializers'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application.rb:352:in `initialize!'
/Users/hashimotoryo/Documents/ruby_clone/stories/config/environment.rb:5:in `<top (required)>'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `require'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `block in require'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:240:in `load_dependency'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/activesupport-4.2.5/lib/active_support/dependencies.rb:274:in `require'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application.rb:328:in `require_environment!'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/gems/railties-4.2.5/lib/rails/application.rb:457:in `block in run_tasks_blocks'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/bin/ruby_executable_hooks:15:in `eval'
/Users/hashimotoryo/.rvm/gems/ruby-2.3.0-dev/bin/ruby_executable_hooks:15:in `<main>'
Tasks: TOP => db:setup => db:schema:load_if_ruby => environment
(See full trace by running task with --trace)
```
When I set up db,(command: rake db:setup) it occurs error.
I think because when comand rake db:setup, devise gem execute model user.rb! So, execute User.import before migrate

### CODE REVIEW

**Comment by kenny-hibino** [comment]
Hey Thank you for contributing! I'd love to have you on the team if you are interested. Could you make a separate branch and push it up?

File: lib/tasks/es.rake [hunk 1]
```diff
@@ -0,0 +1,17 @@
+namespace :es do
+  desc 'recreate index of nescesary model !'
+  task :recreate_index => :environment do
+    [User, Post, Tag].each do |klass|
+      # Delete the previous users index in Elasticsearch
+      klass.__elasticsearch__.client.indices.delete index: klass.index_name rescue nil
+
+      # Create the new index with the new mapping
+      klass.__elasticsearch__.client.indices.create \
+        index: User.index_name,
```
Comment (kenny-hibino) [inline lib/tasks/es.rake]: change this User to klass :)

File: lib/tasks/es.rake [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+        body: { settings: User.settings.to_hash, mappings: klass.mappings.to_hash }
```
Comment (kenny-hibino) [inline lib/tasks/es.rake]: this one too :+1:

File: lib/tasks/es.rake (hunk 1 above)
Comment (Ryohu) [inline lib/tasks/es.rake]: :cry:

File: lib/tasks/es.rake (hunk 2 above)
Comment (Ryohu) [inline lib/tasks/es.rake]: Thank you for reviewing!

**Pull request closed without merging**